composer.json

```diff
@@ -17,10 +17,16 @@
     "require":{
         "php":">4.4.0"
     },
+    "require-dev": {
+        "friendsofphp/php-cs-fixer": "^3.0"
+    },
     "replace":{
         "htmlawed/htmlawed": "*"
     },
      "autoload": {
         "files": ["htmLawed.php"]
+    },
+    "config": {
+        "bin-dir": "bin"
```

Comment:
Why not just keep it in the `vendor/bin` directory?

Reply:
Yeah, we can just keep the original one

Reply:
done